src/blockscoutApi/index.ts

```diff
@@ -49,18 +49,21 @@ export class BlockscoutAPI extends DataSource {
       .catch(this.errorHandling)
   }
 
-  async getEventsByAddress (address: string) {
+  async getEventsByAddress (address: string, flow: string = 'all') {
```

Comment:
Lets make these an `enum` and use it as a type below.

Reply:
Done!